pkg/tsdb/azuremonitor/azuremonitor-resource-handler.go

```diff
@@ -154,9 +155,11 @@ func (s *Service) handleResourceReq(subDataSource string) func(rw http.ResponseW
 		req.URL.Host = serviceURL.Host
 		req.URL.Scheme = serviceURL.Scheme
 
-		rw, err = s.executors[subDataSource].ResourceRequest(rw, req, service.HTTPClient)
+		_, err = s.executors[subDataSource].ResourceRequest(rw, req, service.HTTPClient)
```

Comment:
why is `ResourceRequest` returning a `ResponseWriter`?

Reply:
That's a great question. It doesn't need to but I'd rather not make that change in this PR 😄